bin/env_variables_all.txt

```diff
@@ -53,4 +53,14 @@ IDAM_KEY_DM_STORE=AAAAAAAAAAAAAAAA
 IDAM_KEY_FPL_CASE_SERVICE=AABBCCDDEEFFGGHH
 OAUTH2_CLIENT_CCD_ADMIN=IIIIIIIIIIIIIIII
 APPINSIGHTS_INSTRUMENTATIONKEY=SomeRandomStringForLocalDocker
-DEFINITION_STORE_DB_USE_SSL=false
\ No newline at end of file
+DEFINITION_STORE_DB_USE_SSL=false
```

Comment:
why these vars have been added here?
For example I think `IDAM_BASE_URL=http://idam-api:5000` will override this:
https://github.com/hmcts/ccd-docker/blob/631b59da9bb947b406d6a13d3584fe3c727be062/compose/frontend.yml#L25
hence breaking the Idam Stub feature

Reply:
Those are added for ccg gateway's local execution outside docker.
IDAM_BASE_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}" ==> This line doesn't break because of that change.

Reply:
ok 👍